docs/_extra/api-reference/hypothesis.yaml

```diff
@@ -353,8 +353,9 @@ paths:
             PDF fingerprint.
 
             `*` will match any character sequence (including an empty one), 
-            and a `?` will match any single character. Wildcards are only permitted 
-            within the path and query parts of the URI.
+            and a `?` will match any single character. `*`s are only permitted 
+            within the path and query parts of the URI and `?`s are only permitted
+            within the netloc, path, and query parts of the URI.
```

Comment:
Hmm, although precise, "netloc" is quite a Python-specific term I think (although I'm not sure that it is exclusively used in Python). Perhaps "domain" or "hostname" would be clearer in this context for readers who are not familiar with Python.